### PR TITLE
included prompt,if name, and asserts

### PR DESCRIPTION
**Pull request by nvega23**

### CODE REVIEW

**Comment by nvega23** [comment]
Even Fibonacci sequence

File: Even_Fibonacci.py [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
+    while fib(n) <= n:
+        if fib(n) % 2 != 0:
+            tot += fib(n)
+        n += 1
+    return tot
+
+
+if __name__ == "__main__":
+    tot = fib(5)
+    print(tot)
```
Comment (kswang2400) [inline Even_Fibonacci.py]: nit: you can delete these

File: Even_Fibonacci.py [hunk 2]
```diff
@@ -1,15 +1,32 @@
...
-print(tot)
\ No newline at end of file
+        return fib(n - 1) + fib(n - 2)
+    tot = 0
+    n = 0
+    while fib(n) <= n:
+        if fib(n) % 2 != 0:
+            tot += fib(n)
+        n += 1
+    return tot
```
Comment (kswang2400) [inline Even_Fibonacci.py]: are you sure any of these lines will ever run?

**Review by kswang2400 — CHANGES_REQUESTED** [review]

**Review by kswang2400 — APPROVED** [review]
looks awesome! great job!

**Pull request merged**